lib/Config.ts

```diff
@@ -112,6 +114,10 @@ class Config {
       host: 'localhost',
       port: 5001,
     };
+    this.limits = {
+      lnd: 0.04 ,
+      raiden: 1,
```

Comment:
These should be in the smallest base units (since they get compared to the amounts of the payments which are expressed in these terms), not whole units, and also on a per currency basis. For btc the maximum is 4294967 satoshis, I'm actually not sure about ltc but you should be able to research it. For WETH @moshababo should be able to confirm.